### PR TITLE
COOP/COEP: add META.yml

### PR DESCRIPTION
**Pull request by zcorpan**

### CODE REVIEW

**Review by domenic — COMMENTED** [review]
Please add me :)

**Comment by jugglinmike** [comment]
[`META.yml` files present in all ancestor directories are also considered by the wpt-pr-bot.](https://github.com/web-platform-tests/wpt-pr-bot/blob/0b09f010cccf52c31dfac7e2abbafbc57fe2a1ef/lib/metadata/find-owners.js#L17-L20) Since @domenic, @zcorpan, and @annevk are all listed in [`html/META.yml`](https://github.com/web-platform-tests/wpt/blob/1ecdbc7dd6d5337bf9f9c3942be0260ed4d7746e/html/META.yml), including them here has no effect.

I can image alternate implementations where this *would* make a difference, so including those people here risks misleading others about how the wpt-pr-bot functions. We may want to lint for this.

**Comment by zcorpan** [comment]
How wpt-pr-bot functions might change with https://github.com/web-platform-tests/rfcs/pull/37

But for now I can remove myself and annevk here.

**Review by mikewest — APPROVED** [review]

File: html/cross-origin-opener-policy/META.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+spec: https://gist.github.com/annevk/6f2dd8c79c77123f39797f6bdac43f3e
+suggested_reviewers:
+  - mikewest
+  - jugglinmike
```
Comment (mikewest) [inline html/cross-origin-opener-policy/META.yml]: I'd suggest adding @arturjanc (or @lweichselbaum).

Comment (zcorpan) [reply html/cross-origin-opener-policy/META.yml]: I added both and invited them to the [reviewers team](https://github.com/orgs/web-platform-tests/teams/reviewers). Artur & Lukas, welcome! You may want to [unwatch this repo](https://github.com/orgs/web-platform-tests/teams/reviewers/discussions/1), [filter emails](https://github.com/orgs/web-platform-tests/teams/reviewers/discussions/2) and check the [docs on reviewing tests](https://web-platform-tests.org/reviewing-tests/index.html). 🙂

**Review by annevk — APPROVED** [review]

**Comment by zcorpan** [comment]
Added @hemeryar and @ParisMeuleman per https://github.com/web-platform-tests/wpt/pull/20874#issuecomment-572116047

**Comment by zcorpan** [comment]
@valenting @JuniorHsu would you also like to be added here?

**Comment by valenting** [comment]
Yes, please add me too. Junior is on parental leave this month so he's unlikely to respond very soon.

**Comment by zcorpan** [comment]
@valenting thanks, I've added & invited you.

**Comment by zcorpan** [comment]
I'll merge this now so it starts having an effect. If someone wants to be removed, please comment here (or create a new PR).

Thanks!

**Pull request merged**